src/internal.ts

```diff
@@ -24,8 +24,17 @@ export const isLeft = <E, A>(ma: Either<E, A>): ma is Left<E> => ma._tag === 'Le
 // Record
 // -------------------------------------------------------------------------------------
 
-/** @internal */
-export const hasOwnProperty = Object.prototype.hasOwnProperty
```

Comment:
@OliverJAsh we can just change the name to `has` here

Reply:
Good idea, done!